docs/logs/getting-started-aspnetcore/getting-started-aspnetcore.csproj

```diff
@@ -0,0 +1,14 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
```

Comment:
This property group is not required. All of these properties are already defined in `Common.props` and `Directory.Build.props`.

Reply:
We can remove that in a separate PR.

Reply:
#4583